pwnlib/gdb.py

```diff
@@ -708,6 +708,21 @@ def __getattr__(self, item):
             raise AttributeError()
         return getattr(self.server_breakpoint, item)
 
+    def __setattr__(self, name, value):
+        """Set attributes of the real breakpoint."""
+        if name in (
+            'enabled',
+            'silent',
+            'thread',
+            'task',
+            'ignore_count',
+            'hit_count'
+            'condition',
+            'commands',
+        ):
+            return setattr(self.server_breakpoint, name, value)
```

Comment:
I think this needs to be added to the `Breakpoint` class as well in addition to this `FinishBreakpoint`